target_chains/ton/contracts/contracts/Pyth.fc

```diff
@@ -1,10 +1,36 @@
 #include "imports/stdlib.fc";
 #include "common/errors.fc";
 #include "common/storage.fc";
+#include "common/utils.fc";
+#include "./Wormhole.fc";
+
+const int ACCUMULATOR_MAGIC = 0x504e4155; ;; "PNAU" (Pyth Network Accumulator Update)
+const int MAJOR_VERSION = 1;
+const int MINIMUM_ALLOWED_MINOR_VERSION = 0;
+
+slice verify(slice data) {
```

Comment:
verify is a very generic method name here. you can name it something like `verify_header`

Reply:
good point, will do